File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.js
function solution(s) {
    const string = s.split("");
    const p = string.filter(el=>el==="p" || el==="P").length;
    const y = string.filter(el=>el==="y" || el==="Y").length;
    
    if(p===0 && y===0) return true;
    return p === y ? true:false;
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):
    a = s.count("p") + s.count("P")
    b = s.count("y") + s.count("Y")
    
    if(a == b):
        return True
    elif (a == 0 and b == 0):
        return True
    else:
        return False
